packages/inspector-extension/src/index.ts

```diff
@@ -153,7 +153,7 @@ const consoles: JupyterLabPlugin<void> = {
 
     app.contextMenu.addItem({
       command: CommandIDs.open,
-      selector: '.jp-CodeConsole'
+      selector: '.jp-CodeConsole-promptCell'
```

Comment:
I think we do want context menus on the console area (clear outputs, enable outputs scrolling) so we probably need more complex logic to cover than case.

Reply:
Opps, this is the command for the inspector. Right comments, wrong place in the code.